{
    "cve": {
        "id": "CVE-2024-42988",
        "sourceIdentifier": "cve@mitre.org",
        "published": "2024-10-09T17:15:16.337",
        "lastModified": "2024-10-10T12:51:56.987",
        "vulnStatus": "Awaiting Analysis",
        "cveTags": [],
        "descriptions": [
            {
                "lang": "en",
                "value": "Lack of access control in ChallengeSolves (/api/v1/challenges/<challenge id>/solves) of CTFd v2.0.0 - v3.7.2 allows authenticated users to retrieve a list of users who have solved the challenge, regardless of the Account Visibility settings. The issue is fixed in v3.7.3+."
            },
            {
                "lang": "es",
                "value": "La falta de control de acceso en ChallengeSolves (/api/v1/challenges//solves) de CTFd v2.0.0 - v3.7.2 permite que los usuarios autenticados recuperen una lista de usuarios que han resuelto el desaf\u00edo, independientemente de la configuraci\u00f3n de Visibilidad de la cuenta. El problema se solucion\u00f3 en la versi\u00f3n v3.7.3+."
            }
        ],
        "metrics": {},
        "references": [
            {
                "url": "https://blog.ctfd.io/ctfd-3-7-3/",
                "source": "cve@mitre.org"
            },
            {
                "url": "https://github.com/CTFd/CTFd/pull/2570",
                "source": "cve@mitre.org"
            },
            {
                "url": "https://github.com/CTFd/CTFd/releases/tag/3.7.3",
                "source": "cve@mitre.org"
            }
        ]
    }
}